app/src/main/java/ch/epfl/polybazaar/FillListingActivity.java

```diff
@@ -148,14 +154,10 @@ else if (requestCode == RESULT_LOAD_IMAGE){
                 e.printStackTrace();
                 return;
             }
-
-            stringImage = convertBitmapToString(bitmap);
-            Bitmap resizedBitmap = resizeBitmap(bitmap, 0.5f, 0.5f);
-            stringThumbnail = convertBitmapToStringWithQuality(resizedBitmap, 10);
+            stringImage = convertBitmapToStringWithQuality(bitmap, 10);
```

Comment:
I see that the "10" here appears at least two time. Could you maybe use a constant instead?